lib/net/ldap.rb

```diff
@@ -645,6 +646,13 @@ def get_operation_result
     os
   end
 
+  # Returns the +genPasswd+ from the +PasswdModifyResponseValue+ after
+  # +password_modify+ operation has been called without specifying a new
+  # password.
+  def get_gen_passwd_result
+    @result.extended_response.nil? ? nil : @result.extended_response[0][0]
+  end
```

Comment:
Not a fan of this API. Should we be incorporating extended response results into `get_operation_result`?